src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Please remove the eslint-disable rule 'jsx-a11y/control-has-associated-label' and provide an associated label for interactive controls.